packages/data-broker/src/services/dhis/DhisService.js

```diff
@@ -63,9 +63,6 @@ export class DhisService extends Service {
   };
 
   validatePushData(dataSources, dataValues) {
-    if (dataSources.length !== dataValues.length) {
-      throw new Error('Different number of data sources and values provided to push');
-    }
```

Comment:
We can no longer use this quick check. We could check if any ` dataValue` code does not match to a `dataSource`. Probably this is not required, since warnings will be anyway logged by Pushers if DHIS returns an error.

At any case, using length comparison was a "quick win" and not a full code validation, so I feel it's safe to remove it.